src/pages/SingleCoursePages.js

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect} from "react";
+import { Routes, Route, Link, NavLink, useNavigate, useParams } from 'react-router-dom'
+import apiUtil from '../utils/apiUtil'
+
+//replcae this with the API call. (Finsished)
+//const course = require('./data/courseData.json') 
+
+//localhost:3000/homepage/course/1
+
+//This function is going to be used to display the single course page from when the
+//user selects the course.  
+function DisplayCoursePage(){
+//using useParams() here to grab the number in the course they select then finding the data
+//associated with that course.
+let { paramID } = useParams()
```

Comment:
2 quick hitting points:
1. should be `const` instead of `let`
2. the parameter name contained within the `{}` (in this case you used paramID) has to match the name which is used in the routes. Might I suggest this changes to courseId? That is almost definitely what the parameter will be called in the routing